### PR TITLE
trying to allow users to switch status

### PR DESCRIPTION
**Pull request by g1039**
@hodgestar Please review

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -116,9 +116,16 @@ go.utils = {
     },
 
     update_status: function(im, contact, status) {
-        contact.extra.status = status;
+        contact.extra.status;
```
Comment (hodgestar) [inline src/utils.js]: This needs an `= status`.

**Comment by g1039** [comment]
@hodgestar Ready for re-review

File: go-app-sms.js [hunk 2]
```diff
@@ -124,8 +124,15 @@ go.utils = {
 
     update_status: function(im, contact, status) {
         contact.extra.status = status;
+        var message_set_id = im.config.welcome_message_sets[status];
+        var payload = {
+            contact_key: contact.key,
+            messageset_id: message_set_id,
+            schedule: 1
```
Comment (hodgestar) [inline go-app-sms.js:131]: Hardcoding the schedule here is crazy, but it's what the existing code does, so we can live with it for now.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**